Makefile

```diff
@@ -81,12 +82,15 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build . -t ${IMG}
+	docker build -t ${IMG} --build-arg VERSION_PKG=${VERSION_PKG} --build-arg VERSION=${VERSION} --build-arg VERSION_DATE=${VERSION_DATE} --build-arg OTELSVC_VERSION=${OTELSVC_VERSION} .
 
 # Push the docker image
 docker-push:
 	docker push ${IMG}
 
+cert-manager:
```

Comment:
I couldn't find any instructions on when this make target would be called?

Reply:
Will add to the readme.